### PR TITLE
More bootstrap changes

### PR DESCRIPTION
**Pull request by Dananji**

### CODE REVIEW

File: app/views/playlists/_item_list.html.erb [hunk 1]
```diff
@@ -34,14 +34,23 @@ Unless required by applicable law or agreed to in writing, software distributed
...
   </div>
+  <div id="share-list" class="mt-3 collapse">
+    <ul class="nav nav-tabs share-tabs">
+      <%= render_conditional_partials :share, section:'share-tabs' %>
+    </ul>
+
+    <div class="tab-content">
+      <%= render_conditional_partials :share, section:'tab-content' %>
+    </div>
+  </div>
```
Comment (cjcolvar) [inline app/views/playlists/_item_list.html.erb:53]: I see the `mt-3` class was added to this `<div>`, but I'm curious why did you move this div out of the share partial?

**Review by cjcolvar — COMMENTED** [review]
Wow!  This is a lot of JS/CSS/HTML fixing.  This looks really good to me.  I only had one question.

File: app/views/playlists/_item_list.html.erb (hunk 1 above)
Comment (Dananji) [reply app/views/playlists/_item_list.html.erb:53]: Share partial is rendered within the div with the `col-*` class, which spans the share panel to the width of that space. But it should span the entire width of the row under the share and autoplay buttons.
Do you think we need to render it from a partial instead of putting it in here, because of the condition?

Comment (cjcolvar) [reply app/views/playlists/_item_list.html.erb:53]: Now that you mention it, yes!  :slightly_smiling_face: 
I was thinking that semantically they fit better in the share partial since there is JS in that partial which references them.
So how about pushing the wrapping div into the partial instead of moving the other div out of the partial?  Would that work?

Comment (Dananji) [reply app/views/playlists/_item_list.html.erb:53]: Yes that would work! Let me try that :+1:

**Review by cjcolvar — APPROVED** [review]
Looks great!  Thanks!
